squad/frontend/templates/squad/compare_builds.jinja2

```diff
@@ -36,10 +36,20 @@
               <select id="target-select" name="target" placeholder="{{ _('Enter target version') }}" class="form-control"></select>
             </div>
           </div>
+		  <div class="form-group">
+			<label>{{ _('Comparison type') }}</label>
+			<div>
+			  <input id="test-comparison" name="comparison_type" type="radio" value="test" checked="checked"/>
+			  <label for="test-comparison">{{ _('compare by tests') }}</label>
+			  <br />
+			  <input id="metric-comparison" name="comparison_type" type="radio" value="metric" />
+			  <label for="metric-comparison">{{ _('compare by metrics') }}</label>
+			</div>
+		  </div>
           <div class="form-group">
             <input type="hidden" name="project" value="{{ project.full_name }}" />
             <input onclick="this.form.submit()" value="{{ _('Compare') }}" type="button" class="btn btn-default" />
-            or
+            {{ _('or') }}
```

Comment:
I think we should drop this "or". everybody knows what to do when there is a sequence of buttions

Reply:
agreed.